modtools/create-unit.lua

```diff
@@ -566,6 +587,7 @@ if args.name then
 else
   local unit = df.unit.find(unitId)
   unit.name.has_name = false
+  unit.name.first_name = "" -- remove the first name altogether. gets rid of units having number names.
```

Comment:
I'm not sure I like this change - it results in units with blank names in arena mode, and even without this change in fortress mode, the number names aren't visible in the unit or job lists.
I'm thinking of just changing this back, but is there something I'm missing here?